src/frontend/src/btc/derived/current-address.derived.ts

```diff
@@ -0,0 +1,28 @@
+import {
+	BTC_MAINNET_NETWORK_ID,
+	BTC_REGTEST_NETWORK_ID,
+	BTC_TESTNET_NETWORK_ID
+} from '$env/networks.env';
+import {
+	btcAddressMainnet,
+	btcAddressRegtest,
+	btcAddressTestnet
+} from '$lib/derived/address.derived';
+import { networkId } from '$lib/derived/network.derived';
+import type { OptionBtcAddress } from '$lib/types/address';
+import { nonNullish } from '@dfinity/utils';
+import { derived, type Readable } from 'svelte/store';
+
+export const currentBtcAddress: Readable<OptionBtcAddress | undefined> = derived(
+	[networkId, btcAddressMainnet, btcAddressTestnet, btcAddressRegtest],
+	([$networkId, $btcAddressMainnet, $btcAddressTestnet, $btcAddressRegtest]) => {
+		const mapper: Record<symbol, OptionBtcAddress> = {
```

Comment:
Performance-wise is it more efficient `if` conditions or a mapper? Honest question

Reply:
I don't know, but I don't think it matters much the perfomrnace difference if any in this case.